requirements/pip.txt

```diff
@@ -19,20 +19,20 @@ readthedocs-build<2.1
 # django-tastypie 0.13.x and 0.14.0 are not compatible with our code
 django-tastypie==0.13.0
 
-django-haystack==2.7.0
+django-haystack==2.8.1
```

Comment:
I think this is only used on the API v1, is still worth updating? :thinking:

Reply:
Isn't this used for elasticsearch?

Reply:
No. It's only for v1 API which we should deprecate.